storage/remote/read_handler.go

```diff
@@ -56,10 +56,10 @@ func NewReadHandler(logger *slog.Logger, r prometheus.Registerer, queryable stor
 		marshalPool:               &sync.Pool{},
 
 		queries: prometheus.NewGauge(prometheus.GaugeOpts{
-			Namespace: "prometheus",
-			Subsystem: "api", // TODO: changes to storage in Prometheus 3.0.
-			Name:      "remote_read_queries",
-			Help:      "The current number of remote read queries being executed or waiting.",
+			Namespace: namespace,
+			Subsystem: "remote",
```

Comment:
perhaps make it consistent with `subsystem` being `remote_<client|handler`? (one way or another)

Reply:
you're right, made the change locally but never pushed it.

Reply:
friendly bump @bwplotka